docs/guide/README.md

```diff
@@ -191,7 +191,15 @@ algob -h deploy
 You can write your scripts and tests in TS. To initialize a new typescript project add `--typescript` to the init flag. Example, if you follow the [Create an algob project](#create-an-algob-project) section then:
 
 ```shell
-yarn run algob init --typescript
+yarn run algob init <path-where-to-create> --typescript
+```
+#### Example on `algob init`
+
+You can use below command to initialize the typescript project in `sample-project` folder.
+If `sample-project` folder is not present then it will create one for you. 
```

Comment:
lets add info that we can also do --javascirpt. I think it works the same for both right?

Reply:
If you don't add --typescript then the default is javascript. --typescript will generate typescript files other javascript files.

Reply:
and what happens if you add --javascirpt? works the same as you would add any flag?

Reply:
if we can do it lets add information something like that `Two flags can be used --javascirpt --typescirpt. If no flag was passed the default one is javascirpt...`

Reply:
I have updated the doc.